src/sql/parts/jobManagement/views/jobStepsView.component.html

```diff
@@ -22,4 +22,4 @@ <h1 style="display: inline">Steps</h1>
 		</td>
 	</tr>
 </table>
-<div class='steps-tree' #table style="height: 100%; width: 100%"></div>
\ No newline at end of file
+<div class='steps-tree' #table style="height: 485px; width: 100%"></div>
```

Comment:
where did the size `485px` come from?  Will this work well on various screen resolutions?

Reply:
The 485px is the size from where the steps tree starts to the status bar underneath.

Reply:
How did you determine this is the ideal number?